README.rst

```diff
@@ -99,15 +99,15 @@ Without a Context Manager
 -------------------------
 
 In the example below, we create the `HCSR04` object directly, get the distance every 2 seconds, then
-de-initialize the device.
+de-initialize the device. Note trig and echo should come from the `board` module - they aren't just integers.
 
 ::
 
     from adafruit_hcsr04 import HCSR04
     sonar = HCSR04(trig, echo)
```

Comment:
Thank you for fixing the out of date example in the readme!

Instead of making a note in the paragraph above can you update this line in the example code to show it's usage with the board pins. You can copy it form the example file in the repo if you want: 
https://github.com/adafruit/Adafruit_CircuitPython_HCSR04/blob/master/examples/hcsr04_simpletest.py

You can add `import board` at the top and then change the sonar constructor like this:
```
sonar = HCSR04(trigger_pin=board.D5, echo_pin=board.D6)
```

Reply:
Good idea. I've committed this change.